core/src/main/java/tc/oc/pgm/wool/MonumentWoolFactory.java

```diff
@@ -86,7 +85,7 @@ public String toString() {
         + ", craftable="
         + this.craftable
         + ", visible="
-        + this.visible
+        + this.getShowOptions()
```

Comment:
If I understand this code right this won’t show anything useful right? Should probably just forward the toString to the underlying enum set.

Reply:
ShowOptions just requires a proper toString method for this to work

Reply:
Yes that's what I tried to say hehe